modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -5,255 +5,258 @@
 
 This section describes how to manually upgrade your ownCloud installation.
 
-If you’re not comfortable performing a manual upgrade, you can also use your Linux distribution’s
-xref:installation/linux_installation.adoc[package manager].
-
-[[backup-your-existing-installation]]
-== Backup Your Existing Installation
+== Enable Maintenance Mode
 
-First, xref:maintenance/backup.adoc[backup] the following items:
+Put your server in xref:configuration/server/occ_command.adoc#maintenance-commands[maintenance mode] and *disable xref:configuration/server/background_jobs_configuration.adoc#cron-jobs[Cron jobs]*.
+Doing so prevents new logins, locks the sessions of logged-in users, and displays a status screen so that users know what is happening.
 
-* The ownCloud server data directory
-* The `config.php` file
-* All 3rd party apps
-* The ownCloud server database
+There are two ways to enable maintenance mode.
+The preferred method is to use xref:configuration/server/occ_command#maintenance-commands[the occ command] — which you must run as your webserver user.
```

Comment:
I strongly recommend converting this and the following sentence into a numbered list.